lib/components/button/button.less

```diff
@@ -412,7 +412,7 @@
                 border-color: var(--_bu-bc-hover);
             }
 
-            &:visited:not(:hover):not(:focus) {
+            &:visited:not(:active):not(:focus) {
```

Comment:
Is this change intentional?

Reply:
Yep, that rule is _already_ within a `:hover` selector, so adding `:not(hover)` would make it so the rule here never applies. This is an existing bug that I resolved in the course of this PR.